### DISCUSSION TITLE
`robyn_outputs` failing to produce csv files

### DISCUSSION THREAD

**Discussion by elijahstevensonatfanduel** [facebookexperimental/Robyn#1068] [General]
```
OutputCollect <- robyn_outputs(
  InputCollect,
  OutputModels,
  pareto_fronts = 1, # 1 "auto", # automatically pick how many pareto-fronts to fill min_candidates
  csv_out = "all", # "pareto", "all", or NULL (for none)
  clusters = FALSE, # Set to TRUE to cluster similar models by ROAS. See ?robyn_clusters
  plot_pareto = FALSE, # Set to FALSE to deactivate plotting and saving model one-pagers
  plot_folder = base_output_path, # path for plots export
  export = TRUE # this will create files locally
)
```
Code, above, which previously was working and producing csv files now is not producing csv files. Any idea why this may be happening? Get the generalized message below: `Failed exporting results, but returned model results anyways`.

<img width="811" alt="image" src="https://github.com/user-attachments/assets/6ce297b9-4ad2-49f4-a51b-20a2c8c69b42">

**Comment by sarasbissi** [comment]
Hello  I am having the exact same issue, do you have any solution ?
